src/subdomains/core/sell-crypto/process/buy-fiat.service.ts

```diff
@@ -186,6 +187,7 @@ export class BuyFiatService {
       date: buyFiat.fiatOutput?.outputDate,
       amlCheck: buyFiat.amlCheck,
       isComplete: buyFiat.isComplete,
+      status: TypedHistoryDto.PaymentStatusMapper[buyFiat.isComplete ? 'Complete' : 'Incomplete'],
```

Comment:
`status: buyFiat.isComplete ? PaymentStatus.COMPLETE : PaymentStatus.PENDING` and remove the `Incomplete` entry on the mapper?